audmetric/core/api.py

```diff
@@ -376,6 +376,8 @@ def diarization_error_rate(
             "The truth and prediction "
             "should be a pandas Series with a segmented index conform to audformat."
         )
+    # Warn user if there are no common files
+    _check_common_files(truth, prediction)
```

Comment:
It's not ideal that we need to check twice for this when running calculating DER, one time here and one time inside `identification_error_rate()`